fbarr/src/burn/drivers/misc_pre90s/d_ddragon.cpp

```diff
@@ -214,6 +214,22 @@ static struct BurnDIPInfo Drv2DIPList[]=
 	{0x14, 0x01, 0x80, 0x00, "On"                     },
 	
 	// Dip 2
+	// Note the difficulty settings were pulled from MAME
+	// Do not use the real Double Dragon 2 Manual as
+	// reference, because it lists wrong settings!
+	//
+	// If anyone points you to bad settings from Manual
+	//  they will look like this:
+	//   OFF OFF = Normal
+	//   OFF OFF = Easy
+	//   ON  OFF = Less Than Difficult
+	//   ON  ON  = Difficult
+	{0   , 0xfe, 0   , 4   , "Difficulty"             },

```

Comment:
<img width="555" alt="screen shot 2018-04-01 at 11 43 19 am" src="https://user-images.githubusercontent.com/1490512/38176349-f0131d6e-35a1-11e8-9455-ffcbb08d65e5.png">

Pretty funny to see the original Manual docs :) no proof-reading it seems